utils/sysfs/sysfs.go

```diff
@@ -242,7 +243,11 @@ func (self *realSysFs) GetSystemUUID() (string, error) {
 			//If running on a KVM guest on Power then UUID is /proc/device-tree/vm,uuid
 			id, err = ioutil.ReadFile(path.Join(ppcDevTree, "vm,uuid"))
 			if err != nil {
-				return "", err
```

Comment:
This nested if loop isn't ideal. While you are here, can you refactor the code to not return `id` if any of the if conditions pass?